src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -99,9 +99,9 @@ private static void reloading(CommandSender sender, String what, Object... args)
 	private static void reloaded(CommandSender sender, RedirectingLogHandler r, String what, Object... args) {
 		what = args.length == 0 ? Language.get(CONFIG_NODE + ".reload." + what) : PluralizingArgsMessage.format(Language.format(CONFIG_NODE + ".reload." + what, args));
 		if (r.numErrors() == 0)
-			Skript.info(sender, StringUtils.fixCapitalization(PluralizingArgsMessage.format(m_reloaded.toString(what))));
+			Skript.info(sender, StringUtils.fixCapitalization(PluralizingArgsMessage.format(m_reloaded.toString(what, String.valueOf(r.getTimeTaken())))));
```

Comment:
I'm not sure if the timing information would be of use to the average scripter, definitely not as important as to make it bold / underlined

Reply:
As I and Shane talked about this before [on discord](https://discord.com/channels/135877399391764480/135877399391764480/922896402802151504), it is a good parctice for any scripter to know the time taken to reload a script and might need that to compare between some codes they test + for debugging and reporting issues as well

As for the bold, it is just a style I liked to add to make it pop out

Reply:
(@ShaneBeee to include you in this conversation)

I don't think most scripters care about information like this, most of them just want simple modifications for their server. I'm fine with it being in the reload confirmation message, but don't think it should be highlighted in any way for the reason above.

Reply:
If the matter is with the styling that's fine, I can remove the bold style